inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/group/InlongGroupRequest.java

```diff
@@ -49,8 +49,8 @@ public abstract class InlongGroupRequest extends BaseInlongGroup {
 
     @ApiModelProperty(value = "Inlong group id", required = true)
     @NotBlank(message = "inlongGroupId cannot be blank")
-    @Length(min = 4, max = 100, message = "length must be between 4 and 100")
-    @Pattern(regexp = "^[a-z0-9_.-]{4,100}$", message = "only supports lowercase letters, numbers, '-', or '_'")
+    @Length(min = 4, max = 256, message = "length must be between 4 and 256")
```

Comment:
The design suggestion is to reserve a portion of the buffer for defining the field length in DB, and not fully utilize it to prevent unpredictable errors such as character transcoding.

Reply:
In this case, the field length in the DB can be defined as 256, but the field length in user interaction is limited to 200, which is also a reasonable approach in terms of user understanding.

Reply:
done